### PR TITLE
Adding `link.arsenic.hackclub.com` & `cdn.arsenic.hackclub.com`

### PR DESCRIPTION
**Pull request by SahandSoleimani**
# Adding `link.arsenic.hackclub.com` & `cdn.arsenic.hackclub.com`

## Description

I'm adding subdomains for my hack club for links and also a content delivery network!

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -1142,6 +1146,10 @@ f5qsmeu4qwef4yvo2p6e4a4vrspzdqdc._domainkey.epochvt:
   ttl: 600
   type: CNAME
   value: f5qsmeu4qwef4yvo2p6e4a4vrspzdqdc.dkim.amazonses.com.
+fa:
+  - ttl: 1
+    type: CNAME
+    value: cname.vercel-dns.com.
```
Comment (alx-alexpark) [inline hackclub.com.yaml:1152]: What is this record?

Comment (SahandSoleimani) [reply hackclub.com.yaml:1152]: Hi, I hope you're having a wonderful day!

It's for the Persian Translation of Hack Club that I've made.
You can view my pull request here: https://github.com/hackclub/global/pull/116

Comment (SahandSoleimani) [reply hackclub.com.yaml:1152]: In the readme it mentioned that to get the subdomain going I'd have to create a pull request in the dns repository!

Comment (SahandSoleimani) [reply hackclub.com.yaml:1152]: image

Comment (alx-alexpark) [reply hackclub.com.yaml:1152]: Could you make this into two separate pull requests? One for link and CDN, and one for your translation?

Comment (SahandSoleimani) [reply hackclub.com.yaml:1152]: I now separately only opened a pull request for `cdn.arsenic.hackclub.com`, but I don't know how to open a separate pull request.

Do I have to wait until the first one gets merged then create a new one?

**Pull request closed without merging**